mlir/lib/Dialect/Rock/Transforms/AffixTuningParameters.cpp

```diff
@@ -105,7 +105,9 @@ void AffixTuningParameters::affixTuningParametersImpl(
     perfConfig = perfConfigAttr.getValue().str();
   }
   GemmFeatures features = op.getGemmFeatures();
-  if (rock::isAccel(features, op.getAType())) {
+  bool isAccel =
+      bitEnumContainsAny(features, GemmFeatures::wmma | GemmFeatures::mfma);
```

Comment:
This applies to lots of places. I think it has appeared enough times to justify a dedicated function

```cpp
bool isAccel(GemmFeatures features){
   return bitEnumContainsAny(features, GemmFeatures::wmma | GemmFeatures::mfma);
}
```

So you'd just need to call this function instead of having the expand out the specific features everywhere.